src/app/utils/api.ts

```
@@ -1,20 +1,19 @@
+    'Content-Type': 'application/json',
   },
 });
 
 export const setAuthToken = (token: string | null) => {
-  console.log("Setting auth token:", token ? "Token exists" : "No token");
   if (token) {
-    apiClient.defaults.headers.common["Authorization"] = `Bearer ${token}`;
+    apiClient.defaults.headers.common['Authorization'] = `Bearer ${token}`;
   } else {
-    delete apiClient.defaults.headers.common["Authorization"];
+    delete apiClient.defaults.headers.common['Authorization'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)